framework_tvb/tvb/interfaces/web/controllers/spatial/region_stimulus_controller.py

```diff
@@ -175,7 +176,7 @@ def step_2(self):
         template_specification = dict(title="Spatio temporal - Region stimulus")
         template_specification['mainContent'] = 'spatial/stimulus_region_step2_main'
         template_specification['next_step_url'] = '/spatial/stimulus/region/step_2_submit'
-        template_specification['regionStimSelectorForm'] = region_stim_selector_form
+        template_specification['regionStimSelectorForm'] = FlowController().get_template_dict(region_stim_selector_form)
```

Comment:
Maybe would be nicer to move the get_template_dict() method to BaseController, thus we could use it with self.get_template_dict() and won't be necessary to import FlowController in the other controllers.

Reply:
In fact there is one `def get_template_dict(self, adapter_form)` in the base class SpatioTemporalController inherited also here.
What is the difference between the two methods? I see the decorator is different and the Jinja template ... but did not looked further

Reply:
if we need both, then we will need for one a different name.

Reply:
Indeed, we need both for now. The template here is slightly different. 
The main difference is that we need to send more values at rendering time, while in FlowController we send only the form. Mostly because of the dynamic subforms.
This can be simplified and will be part of task TVB-2617.

For now, I propose to use a different name for method get_template_dict in SpatioTemporalController and move get_template_dict from FlowController to BaseController.
Thanks for pointing this out.

Reply:
I just did this change myself, and could not help to rename both methods :-)